Gible.Web/Components/Pages/Admin.razor

```diff
@@ -0,0 +1,25 @@
+@page "/admin"
+@rendermode InteractiveServer
+@using Knox.Mediation;
+@using Gible.Domain.Commands;
+
+<h1>Admin</h1>
+<div class="row">
+    <div class="col">
+        <button class="btn btn-primary" @onclick="IngestRecipes">Ingest Recipes</button>
+    </div>
+</div>
+
+@code {
+    [Inject]
+    public IMediator Mediator { get; private set; } = default!;
+
+    public async Task IngestRecipes()
+    {
+        var output = "C:\\Users\\noahg\\source\\repos\\Gible\\Gible.Web\\bin\\Release\\net8.0\\publish\\wwwroot\\Processed";
```

Comment:
I think putting images in the wwwroot will wipe the images when it's published again, so this isn't a great solution but it prevents me from having to get an S3 bucket for the project. All images are saved in an archival hard drive anyway so it's not like I'm going to lose them